### ISSUE TITLE
Adding single data - point of interest with an X and Y intercept

### ISSUE THREAD

**Issue by 7elysium** [willdale/SwiftUICharts#84]
I have a 6 line multiline chart. 
I want to plot a single point of data based on user input on this graph to see where that data point lies with respect to the other lines. Preferably highlight it in some manner with X, Y-axis intercepts. 
I tried using yAxisPOI but that only lets me enter in one value (xAxis). And I tried creating another line with 3 points (2 preset ones and entering data into the third middle one) but I couldn't get it to work. 
I have two values [Double, Double] that I want to plot. 

And if this is possible - will the point be appropriately placed along the Y-axis? I'm asking because my Y-axis labels are all string interpolations of integers. 

Thank you so much!

**Comment by willdale** [comment]
Hi, does the request boil done to adding X axis POI?

**Comment by 7elysium** [comment]
Hello!
I think using both an X-axis POI along with a Y-axis together should do the trick. 
But if we do that, then where will the POI label be shown? Won't there be two different POI labels?I don't think it'll be possible to specifically highlight the point where the 2 lines intercept...

This is what I have I'm trying to achieve:
<img width="290" alt="Screen Shot 2021-06-05 at 7 13 42 PM" src="https://user-images.githubusercontent.com/85053130/120893784-a858fa00-c632-11eb-9c73-0d8a9cff1a1c.png">


Also, thank you for looking into this! i really appreciate it!

**Comment by 7elysium** [comment]
Hi Will,

I tried creating two straight lines to get something close to my screenshot using: 
- Line 1: (minimum X, user Y) (user X, user Y), (maximum X, user Y)
- Line 2: (user X, min Y) (user X, user Y), (user X, max Y)
- 
But this approach didn't work.. I tried something similar making one line using yAxisPOI which failed too.

Is there anything else that I can try using the tools already present in the package to get something close to the result I'm going for? 

I'm sorry to bother you with this again, but I really need this for a project. I'm pretty new to coding so I'm not sure what else I can try and all the other charts packages available here don't even come close to yours... 

I'd really appreciate any help!

Thank you so much!
